workloads/kube-burner/common.sh

```diff
@@ -166,17 +166,66 @@ snappy_backup() {
  store_on_elastic
 }
 
-remove_update_taint() {
-  # This is only here until BZ https://bugzilla.redhat.com/show_bug.cgi?id=2035005 gets resolved
-  oc adm taint nodes UpdateInProgress:PreferNoSchedule- --all || true
+get_network_type() {
+if [[ $NETWORK_TYPE == "OVNKubernetes" ]]; then
+  network_ns=openshift-ovn-kubernetes
+else
+  network_ns=openshift-sdn
+fi
+echo $network_ns
 }
 
+check_metric_to_modify() {
+  export div_by=1
+  echo $config | grep -i memory
+  if [[ $? == 0 ]]; then 
+   export div_by=1048576
+  fi
+  echo $config | grep -i latency
+  if [[ $? == 0 ]]; then
+    export div_by=1000
+  fi
+}
+
+run_benchmark_comparison() {
```

Comment:
Thoughts on moving this to a more common place for all workload types?

Reply:
Good point, currently there are 5 workloads which use this function in slightly different way than other.
I will create an issue and we can track it in another PR as it requires modification in other workloads